Source/WebCore/Modules/WebGPU/GPUDevice.cpp

```diff
@@ -68,6 +68,14 @@ namespace WebCore {
 
 WTF_MAKE_ISO_ALLOCATED_IMPL(GPUDevice);
 
+GPUDevice::GPUDevice(ScriptExecutionContext* scriptExecutionContext, Ref<PAL::WebGPU::Device>&& backing)
+: ActiveDOMObject { scriptExecutionContext }
```

Comment:
I think these are normally indented.

Reply:
I'm pretty sure that the initializer list should be indented.